chainstate/test-suite/src/tests/stake_pool_tests.rs

```diff
@@ -241,80 +231,6 @@ fn stake_pool_and_transfer_tokens_same_tx(#[case] seed: Seed) {
     });
 }
 
-#[rstest]
-#[trace]
-#[case(Seed::from_entropy())]
-fn stake_pool_with_tokens_as_input_value(#[case] seed: Seed) {
-    utils::concurrency::model(move || {
-        let mut rng = make_seedable_rng(seed);
-        let mut tf = TestFramework::builder(&mut rng).build();
-
-        let token_issuance_data =
-            random_token_issuance(tf.chainstate.get_chain_config().clone(), &mut rng);
-        let amount_to_issue = token_issuance_data.amount_to_issue;
-
-        // create a tx with coins utxo and token issuance utxo
-        let tx0 = TransactionBuilder::new()
-            .add_input(
-                TxInput::new(
-                    OutPointSourceId::BlockReward(tf.genesis().get_id().into()),
-                    0,
-                ),
-                empty_witness(&mut rng),
-            )
-            .add_output(TxOutput::new(
-                token_issuance_data.into(),
-                OutputPurpose::Transfer(anyonecanspend_address()),
-            ))
-            .add_output(TxOutput::new(
-                OutputValue::Coin(Amount::from_atoms(rng.gen_range(100_000..200_000))),
-                OutputPurpose::Transfer(anyonecanspend_address()),
-            ))
-            .add_output(TxOutput::new(
-                OutputValue::Coin(tf.chainstate.get_chain_config().token_min_issuance_fee()),
-                OutputPurpose::Burn,
-            ))
-            .build();
-        let tx0_id = tx0.transaction().get_id();
-        let token_id = common::chain::tokens::token_id(tx0.transaction()).unwrap();
-
-        let destination = new_pub_key_destination(&mut rng);
-        let (_, vrf_pub_key) = VRFPrivateKey::new_from_rng(&mut rng, VRFKeyKind::Schnorrkel);
-
-        // use token input to stake pool with tokens
-        let tx1 = TransactionBuilder::new()
-            .add_input(
-                TxInput::new(OutPointSourceId::Transaction(tx0_id), 0),
-                empty_witness(&mut rng),
-            )
-            .add_output(TxOutput::new(
-                TokenData::TokenTransfer(TokenTransfer {
-                    token_id,
-                    amount: amount_to_issue,
-                })
-                .into(),
-                OutputPurpose::StakePool(Box::new(StakePoolData::new(
-                    anyonecanspend_address(),
-                    vrf_pub_key,
-                    destination,
-                    0,
-                    Amount::ZERO,
-                ))),
-            ))
-            .build();
-        let tx1_id = tx1.transaction().get_id();
-
-        let result = tf.make_block_builder().with_transactions(vec![tx0, tx1]).build_and_process();
-
-        assert_eq!(
-            result.unwrap_err(),
-            ChainstateError::ProcessBlockError(BlockError::StateUpdateFailed(
-                ConnectTransactionError::TokenOutputInPoSAccountingOperation(tx1_id)
```

Comment:
I think this error is obsolete now

Reply:
Actually this wouldn't work anyway due to accounting, before or now. I'll remove the whole test.

Reply:
You were talking specifically about the error, I'm guessing. I just noticed that I removed the test earlier.